domains/client/block-preprocessor/Cargo.toml

```diff
@@ -23,6 +23,7 @@ sp-blockchain = { version = "4.0.0-dev", git = "https://github.com/subspace/polk
 sp-core = { version = "21.0.0", git = "https://github.com/subspace/polkadot-sdk", rev = "0831dfc3c54b10ab46e82acf98603b4af1a47bd5" }
 sp-domains = { version = "0.1.0", path = "../../../crates/sp-domains" }
 sp-executive = { version = "0.1.0", path = "../../primitives/executive" }
+sp-operator-rewards = { version = "0.1.0", path = "../../primitives/operator-rewards" }
```

Comment:
What do you think of changing this to `sp-domain-block-fees` instead since we are introducing `block_fees` under the ER?